docs/src/pages/[platform]/theming/style-props/examples/SizeExamples.tsx

```diff
@@ -0,0 +1,39 @@
+import { Card, View, Text, useTheme } from '@aws-amplify/ui-react';
+
+// use any CSS value
+export const SizeStylePropExample = () => {
+  return (
+    <View backgroundColor="rgb(131, 140, 149)" padding="1rem" fontSize="3rem">
```

Comment:
This example seem overly complex for demonstrating the size style props. What do you think of simplifying this to just a single View with width and height and background color so you can see it?